packages/ui/src/containers/TermsOfUse/TermsOfUseConfirmModal/index.tsx

```diff
@@ -2,9 +2,12 @@ import React, { ReactNode } from 'react';
 import { useTranslation } from 'react-i18next';
 import reactStringReplace from 'react-string-replace';
 
-import { WebModal, MobileModal } from '@/components/ConfirmModal';
+import { WebModal as ConfirmModal } from '@/components/ConfirmModal';
 import TextLink from '@/components/TextLink';
-import usePlatform from '@/hooks/use-platform';
+
+/**
+ * For web use only confirm modal, does not contain Terms iframe
```

Comment:
So does web open the terms of agreement in a new tab?

Reply:
mentioned in the PR body. under mobile webview, we can't open the terms link in another tab. Also as we need to keep the current session state, can't redirect to the terms page as well. So the best approach is to open a modal on the page for mobile.   As for web, just leave it as a link, simply click and open in the new tab should be good enough.